packages/core-components/api-report.md

```diff
@@ -1163,6 +1176,17 @@ export type SidebarDividerClassKey = 'root';
 // @public
 export const SidebarExpandButton: () => JSX.Element | null;
 
+// @public
+export const SidebarGroup: (
+  props: React_2.PropsWithChildren<SidebarGroupProps>,
```

Comment:
I think `children` should be added to the `SidebarGroupProps` rather than just `SidebarGroup`. Tbh `PropsWithChildren` is a bit meh too imo and you might as well just add a `children?: ReactNode` x)

Reply:
Agree - I tried to replace it now at a few places, which I touched during this PR.